File: src/main/webapp/resources/js/plantainterna/vistaAuditoria/vistaAuditoriaService.js
app.service("vistaAuditoriaService", function ($http) {
    this.consultaAuditoriasVistaAuditoria = function (params) {
        return $http({
            method: "post",
            url: "req/consultaAuditoriasVistaAuditoria",
            data: JSON.stringify(params),
            headers: { 'Content-Type': "application/json; charset=utf-8" },
            transformRequest: angular.identity
        });
    }
    this.consultaDetalleAuditoriaTecnicoVistaAuditoria = function (params) {
        return $http({
            method: "post",
            url: "req/consultaDetalleAuditoriaTecnicoVistaAuditoria",
            data: JSON.stringify(params),
            headers: { 'Content-Type': "application/json; charset=utf-8" },
            transformRequest: angular.identity
        });
    }
    this.consultaDetalleAuditoriaVistaAuditoria = function (params) {
        return $http({
            method: "post",
            url: "req/consultaDetalleAuditoriaVistaAuditoria",
            data: JSON.stringify(params),
            headers: { 'Content-Type': "application/json; charset=utf-8" },
            transformRequest: angular.identity
        });
    }
});

File: src/main/java/com/mx/totalplay/ffm/cloudweb/plantainterna/service/VistaAuditoriaService.java
package com.mx.totalplay.ffm.cloudweb.plantainterna.service;

import com.mx.totalplay.ffm.cloudweb.utilerias.model.ServiceResponseResult;

public interface VistaAuditoriaService {

	public ServiceResponseResult consultaAuditoriasVistaAuditoria(String params);

	public ServiceResponseResult consultaDetalleAuditoriaTecnicoVistaAuditoria(String params);

	public ServiceResponseResult consultaDetalleAuditoriaVistaAuditoria(String params);
}
